### PR TITLE
Add AWS Clair documentation

### PR DESCRIPTION
**Pull request by CarlosRS9**
## Description

This PR adds instructions on how to configure an Amazon Lambda to send Amazon ECR image scan results to CloudWatch Logs and how to use our integration to pull these logs.

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 
- [x] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

File: source/amazon/services/supported-services/ecr-image-scanning.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+.. _amazon_image_scanning:
+
+Amazon ECR Image scanning
+=========================
+
+.. versionadded:: 4.3.0
+
+`Amazon ECR image scanning <https://docs.aws.amazon.com/AmazonECR/latest/userguide/image-scanning.html>`_ uses the Common Vulnerabilities and Exposures (CVEs) database from the open-source `Clair project <https://github.com/quay/clair>`_ to detect software vulnerabilities in container images and provide a list of scan findings, which can be easily integrated into Wazuh thanks to the :ref:`AWS CloudWatch Logs integration <aws_cloudwatchlogs>`.
+
+Amazon ECR sends an event to Amazon EventBridge when an image scan is completed. The event itself is only a summary and does not contain the details of the scan findings. However, it is possible to configure a Lambda function to request the scan findings details and store them in CloudWatch Logs. Here is a quick resume on how the workflow looks like:
```
Comment (noise-kngdm) [inline source/amazon/services/supported-services/ecr-image-scanning.rst:12]: I think that you misused 'resume' which is a false friend. You could use either 'résumé' or 'summary'.
```suggestion
Amazon ECR sends an event to Amazon EventBridge when an image scan is completed. The event itself is only a summary and does not contain the details of the scan findings. However, it is possible to configure a Lambda function to request the scan findings details and store them in CloudWatch Logs. Here is a quick summary of how the workflow looks like:
```

File: source/amazon/services/supported-services/ecr-image-scanning.rst [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+#. An image scan is triggered.
+#. Once the scan is completed Amazon ECR will send an event to EventBridge.
+#. The "Scan completed" event will trigger a Lambda function.
+#. The lambda function will take the data from the "Scan completed" event and will request the scan details.
+#. The Lambda function will create a log group and a log stream in CloudWatch Logs to store the response received.
+#. Wazuh will pull the logs from the CloudWatch log groups using the CloudWatch Logs integration.
+
+This section covers how to configure AWS to store the scan findings in CloudWatch Logs so they can be integrated into Wazuh.
```
Comment (noise-kngdm) [inline source/amazon/services/supported-services/ecr-image-scanning.rst:21]: Consider this rewording.
```suggestion
The following sections cover how to configure AWS to store the scan findings in CloudWatch Logs and how to ingest them into Wazuh.
```

File: source/amazon/services/supported-services/ecr-image-scanning.rst [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+5. Wait until "CREATE_COMPLETE" status is reached. The stack containing the AWS Lambda is now ready to be used.
+
+.. thumbnail:: ../../../images/aws/aws_create_completed.png
+    :title: Stack creation completed
+    :align: center
+    :width: 100%
+
+
+6. Once the Stack configuration is completed this can be tested by manually triggering an image Scan. The scan will result in the creation of a CloudWatch log group called ``/aws/ecr/image-scan-findings/<name of the ECR repository>`` containing the scan results. For every new scan the corresponding log streams will be created inside the log group.
```
Comment (noise-kngdm) [inline source/amazon/services/supported-services/ecr-image-scanning.rst:53]: Consider this rewording.
```suggestion
6. Once the Stack configuration is completed, the Lambda can be tested by manually triggering an image Scan. The scan will result in the creation of a CloudWatch log group called ``/aws/ecr/image-scan-findings/<name of the ECR repository>`` containing the scan results. For every new scan, a corresponding log stream will be created inside the log group.
```

File: source/amazon/services/supported-services/ecr-image-scanning.rst [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+.. thumbnail:: ../../../images/aws/aws_findings2.png
+    :title: Stack creation completed
+    :align: center
+    :width: 100%
+
+
+Wazuh configuration
+-------------------
+
+#. Open the Wazuh configuration file (``/var/ossec/etc/ossec.conf``) and add the following configuration block:
```
Comment (noise-kngdm) [inline source/amazon/services/supported-services/ecr-image-scanning.rst:69]: Consider this rephrasing.
```suggestion
#. Open the Wazuh configuration file -which by default is``/var/ossec/etc/ossec.conf``- and add the following configuration block:
```

**Review by noise-kngdm — CHANGES_REQUESTED** [review]

File: source/amazon/services/supported-services/ecr-image-scanning.rst (hunk 1 above)
Comment (CarlosRS9) [reply source/amazon/services/supported-services/ecr-image-scanning.rst:12]: Done

File: source/amazon/services/supported-services/ecr-image-scanning.rst (hunk 2 above)
Comment (CarlosRS9) [reply source/amazon/services/supported-services/ecr-image-scanning.rst:21]: Done

File: source/amazon/services/supported-services/ecr-image-scanning.rst (hunk 3 above)
Comment (CarlosRS9) [reply source/amazon/services/supported-services/ecr-image-scanning.rst:53]: Every scan will create 4-5 logstreams

**Review by noise-kngdm — APPROVED** [review]
LGTM!

**Review by davidjiglesias — APPROVED** [review]

**Review by MarVergara — CHANGES_REQUESTED** [review]
LGTM from the editorial side after addressing suggested changes.

File: source/amazon/services/supported-services/cloudwatchlogs.rst [hunk 5]
```diff
@@ -1,4 +1,4 @@
-.. Copyright (C) 2019 Wazuh, Inc.
+.. Copyright (C) 2021 Wazuh, Inc.
```
[the remark this answers is not in the document]
Comment (CarlosRS9) [reply source/amazon/services/supported-services/cloudwatchlogs.rst:1]: Done

File: source/amazon/services/supported-services/ecr-image-scanning.rst [hunk 6]
```diff
@@ -0,0 +1,114 @@
+.. Copyright (C) 2021 Wazuh, Inc.
+
```
[the remark this answers is not in the document]
Comment (CarlosRS9) [reply source/amazon/services/supported-services/ecr-image-scanning.rst:2]: Done

File: source/amazon/services/supported-services/ecr-image-scanning.rst [hunk 7]
```diff
@@ -0,0 +1,114 @@
+.. Copyright (C) 2021 Wazuh, Inc.
```
[the remark this answers is not in the document]
Comment (CarlosRS9) [reply source/amazon/services/supported-services/ecr-image-scanning.rst:1]: Done

File: source/amazon/services/supported-services/ecr-image-scanning.rst [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+=========================
+
+.. versionadded:: 4.3.0
+
+`Amazon ECR image scanning <https://docs.aws.amazon.com/AmazonECR/latest/userguide/image-scanning.html>`_ uses the Common Vulnerabilities and Exposures (CVEs) database from the open-source `Clair project <https://github.com/quay/clair>`_ to detect software vulnerabilities in container images and provide a list of scan findings, which can be easily integrated into Wazuh thanks to the :ref:`AWS CloudWatch Logs integration <aws_cloudwatchlogs>`.
+
+Amazon ECR sends an event to Amazon EventBridge when an image scan is completed. The event itself is only a summary and does not contain the details of the scan findings. However, it is possible to configure a Lambda function to request the scan findings details and store them in CloudWatch Logs. Here is a quick summary of how the workflow looks like:
+
+#. An image scan is triggered.
+#. Once the scan is completed Amazon ECR will send an event to EventBridge.
```
[the remark this answers is not in the document]
Comment (CarlosRS9) [reply source/amazon/services/supported-services/ecr-image-scanning.rst:15]: Done

File: source/amazon/services/supported-services/ecr-image-scanning.rst [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+#. The "Scan completed" event will trigger a Lambda function.
+#. The lambda function will take the data from the "Scan completed" event and will request the scan details.
```
[the remark this answers is not in the document]
Comment (CarlosRS9) [reply source/amazon/services/supported-services/ecr-image-scanning.rst:17]: Done

File: source/amazon/services/supported-services/ecr-image-scanning.rst [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+#. The Lambda function will create a log group and a log stream in CloudWatch Logs to store the response received.
```
[the remark this answers is not in the document]
Comment (CarlosRS9) [reply source/amazon/services/supported-services/ecr-image-scanning.rst:18]: Done

File: source/amazon/services/supported-services/ecr-image-scanning.rst [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+#. Wazuh will pull the logs from the CloudWatch log groups using the CloudWatch Logs integration.
```
[the remark this answers is not in the document]
Comment (CarlosRS9) [reply source/amazon/services/supported-services/ecr-image-scanning.rst:19]: Done

File: source/amazon/services/supported-services/ecr-image-scanning.rst [hunk 12]
```diff
@@ -0,0 +1,114 @@
...
+-----------------
+
+AWS provides a `template <https://github.com/aws-samples/ecr-image-scan-findings-logger/blob/main/Template-ECR-SFL.yml>`_ for creating a Stack in CloudFormation that loads the image scan findings from Amazon ECR in CloudWatch using an AWS Lambda function.
+
+How to create the CloudFormation Stack
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Download the ECR Image Scan findings logger `template <https://github.com/aws-samples/ecr-image-scan-findings-logger/blob/main/Template-ECR-SFL.yml>`_ from the official `aws-samples <https://github.com/aws-samples/>`_ GitHub repository.
+
+2. Access `CloudFormation <https://console.aws.amazon.com/cloudformation/home>`_ and click on "Create stack"
```
[the remark this answers is not in the document]
Comment (CarlosRS9) [reply source/amazon/services/supported-services/ecr-image-scanning.rst:34]: Done

File: source/amazon/services/supported-services/ecr-image-scanning.rst [hunk 13]
```diff
@@ -0,0 +1,114 @@
...
+
+3. Create a new Stack using the template from step 1
```
[the remark this answers is not in the document]
Comment (CarlosRS9) [reply source/amazon/services/supported-services/ecr-image-scanning.rst:36]: Done

File: source/amazon/services/supported-services/ecr-image-scanning.rst [hunk 14]
```diff
@@ -0,0 +1,114 @@
...
+
+5. Wait until "CREATE_COMPLETE" status is reached. The stack containing the AWS Lambda is now ready to be used.
+
+.. thumbnail:: ../../../images/aws/aws_create_completed.png
+    :title: Stack creation completed
+    :align: center
+    :width: 100%
+
+
+Once the Stack configuration is completed, the Lambda can be tested by manually triggering an image scan. The scan will result in the creation of a CloudWatch log group called ``/aws/ecr/image-scan-findings/<name of the ECR repository>`` containing the scan results. For every new scan the corresponding log streams will be created inside the log group.
```
[the remark this answers is not in the document]
Comment (CarlosRS9) [reply source/amazon/services/supported-services/ecr-image-scanning.rst:53]: Done

File: source/amazon/services/supported-services/ecr-image-scanning.rst [hunk 15]
```diff
@@ -0,0 +1,114 @@
...
+        <service type="cloudwatchlogs">
+          <aws_profile>default</aws_profile>
+          <aws_log_groups>/aws/ecr/image-scan-findings/name_of_the_ECR_repository</aws_log_groups>
+        </service>
+      </wodle>
+
+    .. note::
+      Check the :ref:`AWS CloudWatch Logs integration <aws_cloudwatchlogs>` to learn more about how the CloudWatch Logs integration works.
+
+#. Restart Wazuh in order to apply the changes:
```
[the remark this answers is not in the document]
Comment (CarlosRS9) [reply source/amazon/services/supported-services/ecr-image-scanning.rst:86]: Done

File: source/amazon/services/supported-services/cloudwatchlogs.rst [hunk 16]
```diff
@@ -74,3 +74,16 @@ Wazuh configuration
...
 
         # service wazuh-agent restart
+
+
+CloudWatch Logs use cases
+-------------------------
+
+Check the :ref:`Amazon ECR Image scanning <amazon_image_scanning>` section to learn how to use the CloudWatch Logs integration to pull logs from Amazon ECR Image scans.
+
+.. topic:: Contents
```
[the remark this answers is not in the document]
Comment (CarlosRS9) [reply source/amazon/services/supported-services/cloudwatchlogs.rst:84]: Done

**Review by MarVergara — APPROVED** [review]

**Pull request merged**